File: Experimental/2023/MattsToys/MattsToy2023/src/main/java/frc/robot/commands/Autos.java
// Copyright (c) FIRST and other WPILib contributors.
// Open Source Software; you can modify and/or share it under the terms of
// the WPILib BSD license file in the root directory of this project.

package frc.robot.commands;

import edu.wpi.first.wpilibj2.command.CommandBase;
import edu.wpi.first.wpilibj2.command.Commands;
import edu.wpi.first.wpilibj2.command.PrintCommand;

public final class Autos {
  /** Example static factory for an autonomous command. */
  public static CommandBase exampleAuto() {
    return Commands.sequence(
        new PrintCommand("Do something"),
        new PrintCommand("Do something else"));
  }

  private Autos() {
    throw new UnsupportedOperationException("This is a utility class!");
  }
}

File: Experimental/2024/Replica of SimBeta4 Practice/src/main/cpp/commands/Autos.cpp
// Copyright (c) FIRST and other WPILib contributors.
// Open Source Software; you can modify and/or share it under the terms of
// the WPILib BSD license file in the root directory of this project.

#include "commands/Autos.h"

#include <frc2/command/Commands.h>

frc2::CommandPtr autos::ExampleAuto(IDrivebase* subsystem) {
  return frc2::cmd::Sequence(
      frc2::InstantCommand([subsystem]() { subsystem->stop(); }).ToPtr(),
      frc2::PrintCommand("Trivial operation....").ToPtr());
}
